test/TestComponent.vue

```diff
@@ -1,3 +1,19 @@
+<script lang="ts">
+import { ref } from 'vue';
```

Comment:
**suggestion**: you may want to add more tests

Reply:
@goester-team23 Added some additional examples. May add more in the future.